cli/installer/docker_compose.go

```diff
@@ -556,19 +557,21 @@ func installDockerDesktop(ui UI) {
 			"Architecture":  detectArchitecture(),
 		},
 		apt: `
-			DISTRO=$(lsb_release -is | tr '[:upper:]' '[:lower:]')
-			# cleanup old installs. See https://docs.docker.com/desktop/install/$DISTRO/#prerequisites
-			rm -rf $HOME/.docker/desktop
-			sudo rm -f /usr/local/bin/com.docker.cli
-			sudo apt-get -y purge docker-desktop
-
 			# add docker repo. See https://docs.docker.com/engine/install/$DISTRO/#set-up-the-repository
 			sudo apt-get -y update
 			sudo apt-get -y install \
 				ca-certificates \
 				curl \
 				gnupg \
 				lsb-release
+
```

Comment:
The variable DISTRO was being resolved to an empty string because `lsb-release` wasn't available. Causing the docker installation to fail the first time, but succeed in the subsequent executions.